unumpy/_multimethods.py

```diff
@@ -784,6 +824,69 @@ def broadcast_to(array, shape, subok=False):
     return (array,)
 
 
+def _normalize_axis(ndim, axis, allow_repeats=False):
+    if isinstance(axis, int):
+        axis = (axis,)
+    if isinstance(axis, (tuple, list)):
+        axis_pos = [ndim + ax if ax < 0 else ax for ax in axis]
```

Comment:
I don't think you need the `if` statement here, and in fact if you don't take the branch `axis_pos` will never be defined and you'll get a name error in the next `if` condition.

Reply:
Yes, I agree.